Frameworks/include/CoreTextInternal.h

```diff
@@ -32,38 +36,47 @@
 
 @interface _CTFrameSetter : NSObject {
 @public
-    idretaintype(_CTTypesetter) _typesetter;
+    StrongId<_CTTypesetter> _typesetter;
 }
 @end
 
 @interface _CTFrame : NSObject {
 @public
     _CTFrameSetter* _frameSetter;
     CGRect _frameRect;
+    StrongId<NSMutableArray> _lines;
+
+    // TODO::
+    // Do we need these anymore?
     CGSize _totalSize;
-    idretaintype(NSMutableArray) _lines;
     std::vector<CGPoint> _lineOrigins;
 }
 @end
 
 @interface _CTLine : NSObject {
 @public
-    NSRange _strRange;
+    CFRange _strRange;
+    CGPoint _lineOrigin;
+    StrongId<NSMutableArray> _runs;
```

Comment:
Lets get the bug on StrongId<> filed.  I am even okay removing StrongID in this case, since it would make the readability a *lot* better in all the other places.

---
In reply to: [77875386](https://github.com/Microsoft/WinObjC/pull/950#discussion_r77875386) [](ancestors = 77875386,77873760,77757580,77757492)